tests/UnifiedSpecTests/command-monitoring/server-connection-id.json

```diff
@@ -98,4 +98,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Please remove these changes from the PR.

Looking at the corresponding files in the specifications repo (e.g. [server-connection-id.json](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/tests/unified/server-connection-id.json)), those _do_ have trailing newlines. This looks like this change may be been introduced by your local environment (not sure if that may be a git configuration or your IDE).